examples/Node/microphone.js

```diff
@@ -20,8 +20,8 @@ detector.on('silence', function () {
   console.log('silence');
 });
 
-detector.on('sound', function () {
-  console.log('sound');
+detector.on('sound', function (buffer) {
+  console.log('sound', buffer);
```

Comment:
Same as above, let's not send the binary data to the console. Let's use a comment here.